README.md

```diff
@@ -92,6 +95,70 @@ Activate Orion's regression detection tool for performance-scale CPT runs effort
 
 Additionally, users can specify a custom path for the output CSV file using the ```--output``` flag, providing control over the location where the generated CSV will be stored.
 
+### Daemon mode
+The core purpose of Daemon mode is to operate Orion as a self-contained server, dedicated to handling incoming requests. By sending a POST request accompanied by a configuration file, users can trigger change point detection on the provided metadata and metrics. Following the processing, the response is formatted in JSON, providing a structured output for seamless integration and analysis. To trigger daemon mode just use the following commands
+
+```
+>> orion daemon-mode
+```
+**Querying a Request to the Daemon Service**
+
+To interact with the Daemon Service, you can send a POST request using `curl` with specific parameters. This allows you to submit a file along with additional information for processing.
+
+*Request URL*
+
+```
+POST http://127.0.0.1:8000/daemon
+```
+
+*Parameters*
+
+- uuid (optional): The uuid of the run you want to compare with similar runs.
+- baseline (optional): The runs you want to compare with.
+
+*Request Body*
+
+The request body should contain the file you want to submit for processing. Ensure that the file is in the proper format (e.g., YAML).
+
+Example
+```
+curl -X POST 'http://127.0.0.1:8000/daemon?uuid=4cb3efec-609a-4ac5-985d-4cbbcbb11625' \
```

Comment:
so, I think preferably the endpoints the users hit are more straightforward. 

This approach expects the caller to provide the uuid and config. I think this is one option for the daemon, but it leaves the opportunity for error. 

I  think a more opinionated approach should be how we initially deploy. For example

` http://orion.perflab:8000/?payload=true&version=4.16` 
this endpoint will then use the payload data we have in our data-warehouse, since that will have the most job runs. We could even drop the `payload` and assume it is set to true by default. 

I think the initial endpoint you are suggesting is something else we can consider (with some adjustments) for more custom change-detection -- however I don't think the http endpoint needs to be like the CLI. For example, we shouldn't have the user post the config via http, I think we have candid configs the user provides are daemon launch.